lang/csharp/src/apache/main/File/Codec.cs

```diff
@@ -140,6 +161,15 @@ public static Codec CreateCodec(Type codecType)
                     return new DeflateCodec();
                 case Type.Null:
                     return new NullCodec();
+                case Type.Snappy:
```

Comment:
@martin-g This is how the codec could be created dynacally, if the proper codec nuget was PackeReferenced